tests/ops/qutrit/test_qutrit_channel_ops.py

```diff
@@ -74,7 +75,7 @@ def test_grad_depolarizing(self, angle):
         dev = qml.device("default.qutrit.mixed")
         prob = pnp.array(0.5, requires_grad=True)
 
-        @qml.qnode(dev)
+        @qml.qnode(dev, diff_method="parameter-shift")
```

Comment:
Added this to ensure test is using the expected method of grad computation.

Reply:
See point above about A vs. F; I thought param shift is an A method?

Reply:
This is a fix to the previous PR [QutritDepolarizingChannel](https://github.com/PennyLaneAI/pennylane/pull/5502), I should have implemented it there, however the function of that PR is unaffected, this ensures the test is testing the correct measurement.